revitAPI/filteredElementCollectorByParameterString.py

```diff
@@ -0,0 +1,82 @@
+'''
+FILTERED ELEMENT COLLECTOR - ALL_MODEL_MARK ELEMENTS ONLY
+'''
+__author__ = 'Adam Bear - adam@ukbear.com'
+__twitter__ = '@adambear82'
+__version__ = '1.0.0'
+
+# Importing Reference Modules
+# CLR ( Common Language Runtime Module )
+import clr
+# Adding the RevitServices.dll special Dynamo module to deal with Revit
+clr.AddReference('RevitServices')
+import RevitServices # Importing RevitServices
+# From RevitServices import the Document Manager
+from RevitServices.Persistence import DocumentManager
+# Adding the RevitAPI.dll module to access the Revit API
+clr.AddReference('RevitAPI')
+import Autodesk # Here we import the Autodesk namespace
+# From the Autodesk namespace - derived down to the Revit Database
+# we import only the required classes
+from Autodesk.Revit.DB import FilteredElementCollector, BuiltInParameter, ParameterValueProvider, ElementId, FilterStringBeginsWith, FilterStringContains, FilterStringEndsWith, FilterStringEquals, FilterStringGreater, FilterStringGreaterOrEqual, FilterStringLess, FilterStringLessOrEqual, FilterStringRule, ElementParameterFilter
```

Comment:
Could we please keep these to max 79 characters per code line as per the Contributing section? At present this line and a couple beyond are sitting at 108 :) 

It simply makes for easier reading and is considered best practise in the Python world under the PEP-8 Style Guide.